ros2param/ros2param/verb/list.py

```diff
@@ -36,6 +36,9 @@ def add_arguments(self, parser, cli_name):  # noqa: D102
         parser.add_argument(
             '--include-hidden-nodes', action='store_true',
             help='Consider hidden nodes as well')
+        parser.add_argument(
+            '--param-prefixes', nargs='?',
```

Comment:
I'm not sure `nargs='?'` is the right choice because it allows an empty `--param-prefixes`

```
$ ros2 param list --param-prefixes
listener:
  bar.baz
  foo
  foo.bar
  foo.bar.baz
```

Is the intent to allow multiple prefixes to be specified? If so, `nargs='+'` or `action='append'` might be better choices.